### ISSUE TITLE
LaTeX Error: File `orcidlink.sty' not found.  

### ISSUE THREAD

**Issue by AtomicNess123** [rstudio/rticles#493]
By filing an issue to this repo, I promise that

- [ x] I have fully read the issue guide at https://yihui.name/issue/.
- [ x] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('rticles')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/rticles')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [ x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

## issue with orcidlink
I downloaded the JSS Rmd template and knitted it. 
This is the output error:
```
output file: MyJSSArticle.knit.md

! LaTeX Error: File `orcidlink.sty' not found.

! Emergency stop.
<read *> 

Error: LaTeX failed to compile MyJSSArticle.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See MyJSSArticle.log for more info.
Execution halted

```

I have followed all advice from the above link unsuccessfully.
Anybody has has this problem?
Thanks!

**Comment by cderv** [comment]
> ! LaTeX Error: File `orcidlink.sty' not found.

It means that [orcidlink](https://www.ctan.org/pkg/orcidlink) CTAN package is missing. You need to install all required package for a compilation to work. 

If you are using TeX Live, installation of missing package should be done automatically by **tinytex**. It seems it is not the case here. Are you using TeX Live, or TinyTeX (https://yihui.org/tinytex/) ?

**Comment by AtomicNess123** [comment]
Thanks for your response! I am not sure which Tex I am using, how can I check? I seem to have both installed, but I am not sure.

**Comment by cderv** [comment]
You can check if you have TinyTeX using `tinytex::is_tinytex()`

You can check if you have TeX Live : `tinytex::tlmgr("--version")`

Otherwise, you can also check what tools is used ` Sys.which("latexmk")` and ` Sys.which("pdflatex")`

**Comment by AtomicNess123** [comment]
Thanks! Here the outputs:
```r
> tinytex::is_tinytex()
[1] FALSE
```

I did `install.package('tinytex')`
But still get `FALSE`.


```r
> tinytex::tlmgr("--version")
tlmgr --version
tlmgr revision 54446 (2020-03-21 17:45:22 +0100)
tlmgr using installation: /usr/local/texlive/2020
TeX Live (https://tug.org/texlive) version 2020
```

```r
> Sys.which("latexmk")
                      latexmk 
"/Library/TeX/texbin/latexmk" 

> Sys.which("pdflatex")
                      pdflatex 
"/Library/TeX/texbin/pdflatex" 
```

**Comment by cderv** [comment]
> is there a command to uninstall TeX Live?

I guess it depends how you installed it. You'll find several way on the web. 

> Also, how would uninstalling TeX Live affect my overall LateX behaviour outside R?

TinyTeX is a normal LaTeX distribution, and installation should put the right tools on the PATH, so outside of R `tlmgr` and others commands should work. TinyTeX can be used also by non R user: https://yihui.org/tinytex/#for-other-users

**Comment by AtomicNess123** [comment]
Thanks, managed to uninstalled and installed in RStudio TinyTeX.
Then I went outside R to compile a PDF document in Latex and got this error:

`! LaTeX Error: File `grffile.sty' not found.`

**Comment by AtomicNess123** [comment]
Thanks. In the meanwhile I am installing TexLive, which is very heavy on disk space but I think it will have all the required packages. When I did tlmgr install *nameofpackage* then there was another package that was missing. I did this several times.  So I think it's best that I install the whole Tex distribution. I must admit I don't know what the differences are between distributions in LateX, it seems a very cryptic domain to me. You talk about installing the full (not so tiny :) TinyTex distribution, but I wonder what the difference is with TeXLive.

**Comment by cderv** [comment]
> You talk about installing the full (not so tiny :) TinyTex distribution, but I wonder what the difference is with TeXLive.

Everything is in here: https://yihui.org/tinytex/ including the maintenance part of installing package for non R user. 

TinyTeX is TeX Live that we bundled in a portable way. It has several advantage including easy install / uninstall. For R Markdown users we bundle a specific version that contains the most common CTAN packages required for R Markdown to work. With time, we added other bundles ([rstudio/tinytex-releases#releases](https://github.com/rstudio/tinytex-releases/#releases)) which increased in sized. The last one is not at all tiny as it contains TeX Live + the full set of CTAN packages as explained in [yihui.org/en/2022/05/tinytex-full](https://yihui.org/en/2022/05/tinytex-full/)

>  In the meanwhile I am installing TexLive, which is very heavy on disk space but I think it will have all the required packages.

I don't know for Tex Live but I know that the full version of TinyTeX will contains every CTAN package. If you want to go with that, it is ~2Go or more.
